internal/server/http.go

```diff
@@ -100,6 +106,22 @@ func (s *HTTPServer) Shutdown() error {
 }
 
 func (s *HTTPServer) HealthCheck(w http.ResponseWriter, request *http.Request) {
+	nextCleanup, err := s.getNextCleanupTime()
+	if err != nil {
+		slog.Error("Failed to get the cleanupTime", slog.Any("err", err))
+		http.Error(w, errorResponse("internal server error"), http.StatusInternalServerError)
+		return
+	}
+
+	commandsLeft, err := s.getCommandsLeft()
+	if err != nil {
+		slog.Error("Failed to get the commands Left", slog.Any("err", err))
+		http.Error(w, errorResponse("internal server error"), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("x-next-cleanup-time", strconv.FormatInt(nextCleanup, 10))
+	w.Header().Set("x-commands-left", strconv.FormatInt(commandsLeft, 10))
```

Comment:
@rishavvajpayee To enable Chrome or other browsers to access certain response headers, you need to include them in the `Access-Control-Expose-Headers` header. This ensures the specified headers are accessible to client-side scripts.